index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE-edge">
+    <meta name="viewpoint" content="width-device-width, initial-scale=1.0">
+    <title>Books</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="/CSS/main.css">
+</head>
+<body>
+
+
+    <h1 class="w-100 text-center">Books</h1>
+    <div class="display-books">
+
+    </div>
+    <div class="mt-3 w-100 text-center">
+        <form action="" class="form w-100 text-center mt-5">
+            <label for="title">Title</label>
+            <br>
+            <input type="text" name="title" id="title">
+            <br>
+            <br>
+            <label for="author">Author</label>
+            <br>
+            <input type="text" name="author" id="author">
+        </form>
+        <p class="btn-danger w-100 text-center mt-5" id="add-button">Add Book</p>
+    </div>
+
+    
+    <script src="/JS/main.js"></script>
```

Comment:
- It looks like you got errors with your URL. I kindly invite you to fix it. Hint: Use GitHub pages to see the bug in an easier way. 🐶 
![image](https://user-images.githubusercontent.com/20563252/124497800-0c312880-dd81-11eb-9dce-c10aafd75d56.png)